### PR TITLE
Implemented Rolit game

### PR DESCRIPTION
**Pull request by AlmazSamatov**

### CODE REVIEW

**Comment by mwunderl** [comment]
Thanks for your contribution! I'll take a look.

**Comment by AlmazSamatov** [comment]
Ok. Good.

File: public/app/scorekeep.js [hunk 1]
```diff
@@ -1,10 +1,11 @@
...
 /* API running on Elastic Beanstalk, frontend running locally or
    on a different domain (CORS) */
-// module.value('api', 'http://XXXXX.elasticbeanstalk.com/api/');
+ //module.value('api', 'http://Scorekeep-env.sdap22vsfv.us-east-1.elasticbeanstalk.com/api/');
 
 /* API running locally on port 5000, frontend on port 8080 (CORS) */
-// module.value('api', 'http://localhost:5000/api/');
\ No newline at end of file
+module.value('api', 'http://loccdсalhost:5000/api/');
+
```
Comment (mwunderl) [inline public/app/scorekeep.js:11]: revert these changes

File: src/main/java/scorekeep/Pair.java [hunk 2]
```diff
@@ -0,0 +1,125 @@
+/*
```
Comment (mwunderl) [inline src/main/java/scorekeep/Pair.java:1]: remove copyright

File: src/main/java/scorekeep/SimpleCORSFilter.java [hunk 3]
```diff
@@ -15,6 +15,8 @@ private static UrlBasedCorsConfigurationSource configurationSource() {
     config.setAllowCredentials(true);
     // Modify allowed origins if you run the client at a different endpoint
     config.addAllowedOrigin("http://localhost:8080");
+    config.addAllowedOrigin("http://localhost:5000");
+    config.addAllowedOrigin("http://Scorekeep-env.sdap22vsfv.us-east-1.elasticbeanstalk.com/api/");
```
Comment (mwunderl) [inline src/main/java/scorekeep/SimpleCORSFilter.java:19]: revert this change

File: src/main/java/scorekeep/Sns.java [hunk 4]
```diff
@@ -16,20 +16,20 @@
...
   public static void createSubscription() {
-    String topicarn = System.getenv("NOTIFICATION_TOPIC");
-    String emailaddress = System.getenv("NOTIFICATION_EMAIL");
-    SubscribeRequest subRequest = new SubscribeRequest(topicarn, "email", emailaddress);
-    snsclient.subscribe(subRequest);
+//    String topicarn = System.getenv("NOTIFICATION_TOPIC");
+//    String emailaddress = System.getenv("NOTIFICATION_EMAIL");
+//    SubscribeRequest subRequest = new SubscribeRequest(topicarn, "email", emailaddress);
+//    snsclient.subscribe(subRequest);
   }
```
Comment (mwunderl) [inline src/main/java/scorekeep/Sns.java:33]: revert these changes

File: src/main/java/scorekeep/WebConfig.java [hunk 5]
```diff
@@ -15,12 +15,12 @@ public Filter SimpleCORSFilter() {
...
-  }
+//  static {
+//    if ( System.getenv("NOTIFICATION_EMAIL") != null ){
+//      try { Sns.createSubscription(); }
+//      catch (Exception e ) {
+//        logger.warn("Failed to create subscription for email "+  System.getenv("NOTIFICATION_EMAIL"));
+//      }
+//    }
+//  }
 }
```
Comment (mwunderl) [inline src/main/java/scorekeep/WebConfig.java:26]: revert these changes

**Review by mwunderl — CHANGES_REQUESTED** [review]
Thanks for your patience. Let me know if you have questions.

**Pull request closed without merging**